File: c++/podzielne.cpp
#include <iostream>

using namespace std;

int main(){
    int poczatek,koniec,dzielnik;

    cin >> poczatek;
    cin >> koniec;
    cin >> dzielnik;

    int podzielne = 0;
    for(int i = poczatek; i <= koniec; i++){
        if(i%dzielnik == 0)
            podzielne++;
    }

    cout << podzielne;
}

File: python/podzielne.py
poczatek, koniec, dzielnik = input().split(' ')
poczatek, koniec, dzielnik =  int(poczatek), int(koniec), int(dzielnik)

podzielne = int((koniec / dzielnik)) - int((poczatek / dzielnik))
if poczatek % dzielnik == 0:
    podzielne += 1

print(podzielne)